fdk-organization-bff.yaml

```diff
@@ -16,13 +16,13 @@ servers:
 paths:
   /organizationcatalogs:
     get:
-      summary: Get all organizations and their content in Felles Datakatalog
+      summary: Get all organizations with content in Felles Datakatalog
```

Comment:
Bedre med:
" Get all organizations that have published content in Felles Datakatalog."

Reply:
3d4c238